### DISCUSSION TITLE
Is it possible to set up a centralized location that offers to make PRs, regardless of repo?

### DISCUSSION THREAD

**Discussion by viacheslauy-flutterint** [community/community#120278] [New to GitHub]
Hello all. My company has recently transitioned from BitBucket to GitHub, and I'm still learning the ropes.

One particular feature that I liked about BitBucket is the way I was offered to make a PR after pushing some changes - the popup with the `Create Pull Request` button would always pop up on the main dashboard of the BitBucket site, regardless of the repo. So if I for example pushed changes to repo `Foo` and repo `Bar`, I would see two small popups for creating a PR for both repos **in one location**.

I am trying to achieve the same level of convinience with GitHub. Currently if I make changes to two separate repos under GitHub (Eg. repos `Foo` and `Bar` from previous example), then I have to explicitly visit each repo page to see this popup: first the `Foo` repo page, click the `Compare & Pull Request` in the popup, and then the `Bar` repo page, and then also click the `Compare & Pull Request` in the popup.

Is there a way to have these popups 'pop up' in one centralized location? Thank you.

**Comment by iqnite** [comment]
[Projects](https://docs.github.com/de/issues/planning-and-tracking-with-projects) are exactly what you're looking for!
